origami.json

```diff
@@ -252,6 +278,19 @@
 				}
 			},
 			{
+				"name": "top-story-video",
```

Comment:
Can this demo also have the `hidden: true` property. We're trying to keep Next specific examples out of the demos where possible